src/geowombat/ml/classifiers.py

```diff
@@ -356,6 +363,12 @@ def predict(
         if y.gw.ntime == 1:
             y = y.sel(time="t1")
 
+        y = (
+            y.chunk({"band": -1, "y": data.gw.row_chunks, "x": data.gw.col_chunks})
```

Comment:
Maybe a comment here that this is converting the `numpy` array to a `dask` array.